portfolio/src/main/webapp/index.html

```diff
@@ -97,16 +108,20 @@ <h5 class="card-title text-primary">Fighter Game</h5>
           </div>
         </div>
         <div class="col-md-4 mb-2">
-          <div class="card h-100">
+          <div class="project-card card h-100">
+            <img src="./images/maze-ant.gif" alt="Image of the ant moving through the maze" class="freezeframe card-img-top">
             <div class="card-body">
               <h5 class="card-title text-primary">Maze Ant</h5>
-              <p class="card-text">A little triangle ant guy that uses his raytracing "antennae" to traverse random mazes!
+              <p class="card-text">A little ant guy that uses his raytracing "antennae" to traverse random mazes!
               He learned to do this with my modular genetics library!</p>
             </div>
           </div>
         </div>
       </div>
 
+      <!-- Enable Freezeframe -->
+      <script src="./js/freeze.js"></script>
```

Comment:
Just wondering why this script isn't in the header as well?

Reply:
It's not in the header because the Freezeframe.js module relies on the gifs being present in the DOM when it is instantiated. Is this bad practice? I could add it to the body's onload event.

Reply:
Yeah I think I'd slightly prefer it that way, we can have the script still in `<head>` and call the function in onload.

Reply:
Yeah that makes sense. Keeps everything in the same place and keeps some uniformity: I'm already using onload so why not use it again.